### ISSUE TITLE
Approval required

### ISSUE THREAD

**Issue by jsxyhelu** [vardanagarwal/opencv_python#1]
I read your blog
https://medium.com/dataseries/designing-image-filters-using-opencv-like-abode-photoshop-express-part-1-8765e3f4495b
feels good. I have several similar PS functions too. Have you considered submitting these codes to OpenCV? This is cool!
 If you are willing to do this job, I can provide cooperation, just I have some experience in this area. If you are not willing to do it, then I will submit it myself, and I will sign your name.

**Comment by vardanagarwal** [comment]
Yes I am willing to submit them, but I have no idea how to do this. Could you please tell me more how to do this.

**Comment by jsxyhelu** [comment]
something like this
https://github.com/opencv/opencv/pull/17274

I suggest that you first submit pr with the existing content, and then we will convince members to merge。

**Comment by vardanagarwal** [comment]
There is a problem. I read the guidelines and stuff and they want the program in C++ which I don't know. If you want you can covert it to C++ and both of us can take half the credit for this work if that possible.

**Comment by jsxyhelu** [comment]
OK,I will try my best.
